pinot-core/src/main/java/org/apache/pinot/core/indexsegment/mutable/MutableSegmentImpl.java

```diff
@@ -611,15 +646,18 @@ private int getOrCreateDocId(Map<String, Object> dictIdMap) {
 
     // All metric columns should have no-dictionary index.
     // All metric columns must be single value
```

Comment:
I don't fully understand the statements here if I didn't read the exact code. Can we add some comments on the description of this method to say aggregate metric is only supported for use cases when all the columns are single value?

Reply:
The comments at the head of the method specify the limitations.
The additional limitatation that a metric must be single value is not specified, I can move the comment from insie the method to outside of it.
I am not sure what you want me to change/improve here

Reply:
moved comments around